test/main.sh

```diff
@@ -499,12 +499,12 @@ if [ "${1:-"all"}" != "cluster" ]; then
     run_test test_lxd_benchmark_basic "lxd-benchmark basic init/start/stop/delete"
     run_test test_shutdown "lxd shutdown sequence"
     run_test test_snapshots "container snapshots"
-    run_test test_snap_restore "snapshot restores"
-    run_test test_snap_expiry "snapshot expiry"
-    run_test test_snap_schedule "snapshot scheduling"
-    run_test test_snap_volume_db_recovery "snapshot volume database record recovery"
-    run_test test_snap_fail "snapshot creation failure"
-    run_test test_snap_multi_volume "multi-volume snapshots"
+    run_test test_snapshot_restore "snapshot restores"
+    run_test test_snapshot_expiry "snapshot expiry"
+    run_test test_snapshot_schedule "snapshot scheduling"
+    run_test test_snapshot_volume_db_recovery "snapshot volume database record recovery"
+    run_test test_snapshot_fail "snapshot creation failure"
+    run_test test_snapshot_multi_volume "multi-volume snapshots"
```

Comment:
This is preparation work to have the `test_snap_` prefix reserved for snap-based tests.